### ISSUE TITLE
Radial chord selection zones

### ISSUE THREAD

**Issue by zachkinstner** [aestheticinteractive/Guitar-VR#7]
Discussed in the [DevUp 2015-10-14](https://youtu.be/iGT_iR5DGRo?list=PLmUZ4l5SqKXxVdI8GNld1-YZRClvu-ljw) video.

The idea is to have a single row of chord selectors, which follow the notes/layout of a guitar's fretboard. The left hand's cursor could be in the palm, instead of the fingertip, which might make it possible to rotate/pose your left hand without moving the cursor out of the selected strumming zone.

**Comment by zachkinstner** [comment]
One concern is that the selectors might be too thin along the left-right direction. They would have a much larger height and depth, however, so maybe that is enough to make up for it. Players would have pretty large freedom of movement (without changing the chord) in two out of three dimensions.

**Comment by zachkinstner** [comment]
From @zachkinstner in a Reddit message:

> I'm guessing that [the chords will] be aligned in a single row, following the pattern of a typical fretboard. It might feel a bit like sliding to a note on a trombone.
> 
> With a linear arrangement like that, it becomes easier to re-order or even exclude certain notes. Those probably aren't near-term features, though. For example, players could choose to show only the chord selectors for G, C, and D, which makes the chord selectors larger, and makes it harder to choose an incorrect chord while playing.

**Comment by zachkinstner** [comment]
Change of plans -- I decided on a radial layout. Here is the original mockup:

![Radial Layout Mockup](https://cloud.githubusercontent.com/assets/3940764/10616530/86cd70ea-7733-11e5-93dc-efa176e659b6.png)

What I like about the radial layout is that you never need to move through any chord selector to get to another one -- you can travel through the center instead. The selected chord would "lock" in place until you select another one.

**Comment by zachkinstner** [comment]
This is implemented in my local build, and working nicely. I started out with the 3D-wireframe design, but found that there was too much "visual stuff" there to be functional. Instead, I kept the (invisible) 3D shape for the "selection zone" functionality, but flattened the visual design to be simple panels. The panels each have a visual indicator that grows from the inner side to the outer side as you get closer to it.
